src/main/java/cubicchunks/server/CubeProviderServer.java

```diff
@@ -403,6 +405,18 @@ private void populateCube(Cube cube) {
                 popcube.setPopulated(true);
             }
         });
+        
+        if (CubicChunks.Config.BoolOptions.USE_VANILLA_CHUNK_WORLD_GENERATORS.getValue()) {
```

Comment:
Populators for vanilla may assume there always is terrain already generated in the population area, so they may crash if it's not.